portal/templates/portal/base_old.html

```diff
@@ -53,6 +53,8 @@
     </script>
     {% endblock check_user_status %}
 
+    <script src="https://www.google.com/recaptcha/api.js" async defer></script>
```

Comment:
wait do we still use base_old 🙀

Reply:
I don't know... but it's there so I updated it

Reply:
👍